docs/Running-Simulations-Tutorial/Running-a-Single-Node-Simulation.rst

```diff
@@ -108,9 +129,10 @@ have a NIC. This hardware configuration models a Quad-core Rocket Chip with 4
 MB of L2 cache and 16 GB of DDR3, and **no** network interface card.
 
 We will leave the last section (``[workload]``) unchanged here, since we do
-want to run Linux on our simulated system. The ``terminateoncompletion``
-feature is an advanced feature that you can learn more about in the
-:ref:`manager-configuration-files` section.
+want to run the buildroot-based Linux on our simulated system (you could also
+change this to fedora-uniform.json if you'd like to boot fedora). The
```

Comment:
I don't want to point to fedora here because it's good to have the intro tutorials be as one-track as possible.